CHANGELOG

```diff
@@ -5,6 +5,12 @@ title: Details
 
 All notable changes to this project will be documented in this file.
 
+## security
+
+### 2020-07-30 v0.0.59
+Added
+- Notification callback after changes in policies / users / attributes
+
 ## scheduler
 
 ### 2020-07-23 v0.0.10
```

Comment:
Move that information to ## scheduler section line 91. We suppose to have one section with dates and changes for one service :)

Reply:
Done